src/components/Txt/index.tsx

```diff
@@ -48,22 +64,63 @@ export const align = style({
 	cssProperty: 'text-align',
 });
 
+const Copy = styled(Box)<{ showCopyButton: 'always' | 'hover' | undefined }>`
+	cursor: pointer;
+	visibility: ${(props) =>
+		props.showCopyButton === 'always' ? 'visible' : 'hidden'};
+	&:hover {
+		visibility: visible;
+	}
+	display: inline;
+`;
+
+const copy = (props: ThemedTxtProps) =>
+	props.copy &&
```

Comment:
Let's make this check a boolean

```suggestion
	!!props.cop &&
```